capsules/src/net/udp/udp_port_table.rs

```diff
@@ -32,15 +32,16 @@
 //! userspace UDP driver to check which ports are bound, and vice-versa, such that
 //! exclusive access to ports between userspace apps and capsules is still enforced.
 
+use crate::net::network_capabilities::{NetworkCapability, UdpVisibilityCapability};
 use core::fmt;
 use kernel::capabilities::{CreatePortTableCapability, UdpDriverCapability};
 use kernel::common::cells::{OptionalCell, TakeCell};
-use kernel::ReturnCode;
+use kernel::ReturnCode; // TODO: remove debug
 
 // Sets the maximum number of UDP ports that can be bound by capsules. Reducing this number
 // can save a small amount of memory, and slightly reduces the overhead of iterating through the
 // table to check whether a port is already bound.
-pub const MAX_NUM_BOUND_PORTS: usize = 5;
+pub const MAX_NUM_BOUND_PORTS: usize = 16;
```

Comment:
Add a comment that if this number is changed the kernel tests relying on it will fail